crates/edr_evm/src/lib.rs

```diff
@@ -65,3 +60,8 @@ pub mod interpreter {
 pub mod precompile {
     pub use revm::precompile::{u64_to_address, PrecompileSpecId, Precompiles};
 }
+
+// /// Primitive types of Ethereum
+// pub mod primitives {
+//     pub use revm::primitives::*;
+// }
```

Comment:
Is this intentional?

Reply:
No it was not; thanks!

Reply:
Addressed in https://github.com/NomicFoundation/edr/pull/529/commits/255137ab45eafcb0023ba8fc16515cef14edfdbb